src/lib/containers/table/datasets/DatasetItemsEditor.tsx

```diff
@@ -104,13 +106,15 @@ export function DatasetItemsEditor(props: DatasetItemsEditorProps) {
 
   const mutation = useUpdateEntity<Dataset>({
     onSuccess: () => {
-      displayToast(
-        'Create a Version of this Dataset to freeze it in its current state',
-        'success',
-        { title: 'Dataset Saved' },
-      )
       if (onSave) {
         onSave()
```

Comment:
We will specify the new onSave toast in SWC, which will be shown when the `onSave` callback is invoked.